react/features/whiteboard/actionTypes.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Disable the whiteboard functionality.
+ * {{
+ *      type: DISABLE_WHITEBOARD
+ * }}
+ */
+export const DISABLE_WHITEBOARD: string = 'DISABLE_WHITEBOARD';
+
+/**
+ * Enable the whiteboard functionality.
+ * {{
+ *      type: ENABLE_WHITEBOARD,
+ *      id,
+ *      participantId,
+ *      collabLink
+ * }}
+ */
+export const ENABLE_WHITEBOARD: string = 'ENABLE_WHITEBOARD';
+
+/**
+ * Toggle the whiteboard functionality.
+ * {{
+ *      type: TOGGLE_WHITEBOARD,
+ *      id,
+ *      participantId
+ * }}
+ */
+export const TOGGLE_WHITEBOARD: string = 'TOGGLE_WHITEBOARD';
+
+/**
+ * Updates the username status.
+ * {{
+ *      type: SET_USERNAME_STATUS,
+ *      elements
+ * }}
+ */
+export const SET_USERNAME_STATUS: string = 'SET_USERNAME_STATUS';
```

Comment:
This sounds very generic, what does it do?

Reply:
It helps in updating the whiteboard collaborator name when the jitsi display name is changed. I agree it's generic, I'll try to find a better name.

Reply:
Why do we need it tho? Can't we just set it the first time unconditionally, and then a useEffect can take care of updating it if the local name has changed?

Reply:
You mean to do this in an useEffect in the displayName input component? Left a comment on the whiteboard component about this.